src/Microsoft.Diagnostics.Monitoring.Options/CollectionRuleStateReasons.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    public class CollectionRuleStateReasons
+    {
+        public const string Running = "This collection rule is active and waiting for its triggering conditions to be satisfied.";
```

Comment:
Move these to a resource, to make it easier to localize if we choose to in the future.

Reply:
This might be the correct option, but one thing I should mention is that we're accessing some of these values in FunctionalTests, which I don't believe will be able to access the Strings.resx file. That being said, when I encountered this issue previously, we just added the necessary constants in the FunctionalTests file and left a comment saying that they need to be identical to the ones in Strings.resx

Reply:
Another option is not test the reason in the functional tests if validating the state is sufficient enough.